### DISCUSSION TITLE
Why error when to many CSS files?

### DISCUSSION THREAD

**Discussion by NIKL45** [BCsabaEngine/svelteesp32#32] [Q&A] [answered]
Hello there,
I just wanted to ask why you are throwing an error message in your demo project when there is more than one CSS file. 
Is that specific to your demo project, or is this a general “advice”?

https://github.com/BCsabaEngine/svelteesp32/blob/e43587cbdc8cd02a3f623ec3caaff1720a6b0321/demo/esp32/src/main.cpp#L51-L53

**Comment by BCsabaEngine** [comment, marked as answer]
Hello!

I am very glad that you are examining the project in such detail.
Checking the number of file types is an option that you can use to protect against "accidental error". If you have a development setup that requires only one CSS file. But if one of the frontend colleagues accidentally generates another CSS file (using a wrong method), then our build process will indicate an error.

This is just an example usage that may be useful, but not intended to be followed. However, in our small team, the PNG files are set to 10, because images that we don't notice can "run up" by mistake.

So it's just an example, not a recommendation.

**Reply by NIKL45** [reply]
Thank you very much for that detailed and quick response!
